app/assets/stylesheets/application.scss

```diff
@@ -1,7 +1,3 @@
-strong {
```

Comment:
This used to be needed for the tasklist but is no longer needed so best to remove, plus it's throwing off the formatting for this page

Reply:
Yes, it was added to all strong tags in the service. I have checked all pages with 'strong' and they all look as they should. Anthony has ACd